index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="he">
+<html lang="he" dir="rtl">
```

Comment:
This would also flip the sidebar in the English mode as a side effect
But it is not a big deal since the target audience are people that speak Hebrew